README.md

```diff
@@ -122,6 +122,41 @@ You can build a docker image locally with the following commands:
 
 *NB* if you are on a Mac, you will need [gnu-tar](https://formulae.brew.sh/formula/gnu-tar).
 
+## Using Prometheus as a Go Library
+
+### Remote Write
+
+We are publishing our Remote Write proto independently at
```

Comment:
Perhaps proto → protobuf ? "proto" alone isn't necessarily telling the reader that we are dealing with Google protocol buffers.